README.md

```diff
@@ -62,6 +71,47 @@ To run GitGlance locally, follow these steps:
 ## Contributing
 Contributions are welcome! If you encounter any issues or have suggestions for improvements, please feel free to open an issue or submit a pull request.
 
+
+## How to Contribute
+We welcome contributions from the community to enhance GitGlance further. Here's how you can contribute:
+
+# 1. Fork the Repository
```

Comment:
I think these headings are too big. They would be better suited as bullet points instead.

![Screenshot at 2024-05-12 15-43-04](https://github.com/devXprite/gitglance/assets/80192140/dc98c73d-9b1f-4f28-b5aa-0c10467c5e2c)